homeassistant/components/automation/__init__.py

```diff
@@ -521,6 +523,23 @@ def started_action():
                     await self.action_script.async_run(
                         variables, trigger_context, started_action
                     )
+            except ServiceNotFound as err:
```

Comment:
This removes the logging, as previously done below. However, the script itself already logs this too. So, previously, unknown services would show up in the logs twice.